templates/repo/diff/compare.tmpl

```diff
@@ -184,23 +184,18 @@
 		{{end}}
 	{{else if and .PageIsComparePull (gt .CommitCount 0)}}
 		{{if .HasPullRequest}}
-			<div class="ui segment grid title">
-				<div class="twelve wide column issue-title">
-					{{ctx.Locale.Tr "repo.pulls.has_pull_request" (print $.RepoLink "/pulls/" .PullRequest.Issue.Index) $.RepoRelPath .PullRequest.Index}}
-					<h1>
-						<span id="issue-title">{{RenderIssueTitle $.Context .PullRequest.Issue.Title ($.Repository.ComposeMetas ctx)}}</span>
+			<div class="existing-issue-block ui segment tw-flex tw-items-center tw-space-between tw-p-3 tw-gap-3">
+				<div class="tw-flex tw-flex-1 tw-gap-3 tw-items-center gt-ellipsis">
```

Comment:
Why gt-ellipsis here? It doesn't work with `flex` (IIRC we have discussed it in another PR).

I think this `gt-ellipsis` could be removed, because the children elements could have proper gt-ellipsis

Reply:
What is the proper fix? I found using `gt-ellipsis` two times works reliable.

Reply:
There is already `<div class="issue-title gt-ellipsis">` below, why this "flex" needs `gt-ellipsis`? Flex doesn't work with it ....

Reply:
`min-width: 0` on the parent works, but I still need `gt-ellipsis` in the child to be ellipsed.